sources/osg/polyfill.js

```diff
@@ -63,4 +63,19 @@
         }
         return set;
     };
+
+    // IE11 doesn't support Event() constructor
+    // see https://stackoverflow.com/questions/26596123/internet-explorer-9-10-11-event-constructor-doesnt-work
+    if (typeof Event === 'object') {
+        function CustomEvent(event, params) {
+            params = params || { bubbles: false, cancelable: false, detail: undefined };
+            var evt = document.createEvent('CustomEvent');
+            evt.initCustomEvent(event, params.bubbles, params.cancelable, params.detail);
+            return evt;
+        }
+
+        CustomEvent.prototype = window.Event.prototype;
+
+        window.Event = CustomEvent;
```

Comment:
Better not change Event browser native ?
I thought we would try
- no override of Event 
- usage of "new CustomEvent" instead of "new Event" in rest of the code (for all browser, not just IE)

Reply:
Mostly agree.

Reply:
I don't like the idea of changing working code for all browsers because of a workaround for an old messed up browser.
With this change the input manager code stays the same, only IE11 is affected.

Reply:
I tends to agree with nehon on this point, I find it best if ugly polyfill code is contained in a single file (as much as possible).

Reply:
rationale is that after reading customEvent is not much different than Event, just better (you can add data, not just an event name)
All code can do "new CustomEvent" without spreading polyfill or doing bad code for others

Reply:
agree with @Nehon it will override only on ie11 so good for me. If later we really need CustomEvent then we will be able to change for a good reason